src/components/layout/footer.js

```diff
@@ -367,9 +392,6 @@ const Footer = ({ has_banner_cookie }) => (
                             <Link to="/partners/payment-agent">{localize('Payment agents')}</Link>
                         </div>
                     </Col>
-                    <Col margin_top width="25%">
-                        {!isProduction() && <LanguageSwitcher />}
```

Comment:
why are we removing language switcher here?, please add it back, this is to help translator switching language 🙏